compiler_opt/rl/regalloc/regalloc_runner.py

```diff
@@ -74,9 +76,11 @@ def _compile_fn(
 
     input_ir_path = cmd_path = thinlto_index_path = None
     if len(file_paths) == 3:
-        input_ir_path, cmd_path, thinlto_index_path = file_paths
+      input_ir_path, cmd_path, thinlto_index_path = file_paths
+    elif len(file_paths) == 2:
+      input_ir_path, cmd_path = file_paths
     else:
-        input_ir_path, cmd_path = file_paths
+      logging.fatal("Expected 2 or 3 file paths")
```

Comment:
nit: could you use '' instead of "" for the consistency of the file, otherwise lgtm

thanks for the fix!

Reply:
I just changed it to `"Instead of 2 or 3 file paths, got unexpected value"`. Let me know if you want me to change it from this.

Reply:
oh, i just mean use single quote '' instead of double quote "" for the string message cause the other places in this files are all using single quote '' (see line 72 and 73 above).

Sorry for the confusion -_-!

Reply:
That makes so much more sense. Sorry for the confusion. Should be fixed in the latest version.